src/Sarif.Driver.UnitTests/SarifHelpers.cs

```diff
@@ -23,17 +23,17 @@ internal static class SarifHelpers
                 resultAction(result);
             }
 
-            if (run.ToolNotifications != null)
+            if (run.Invocation.ToolNotifications != null)
```

Comment:
Any need to check for `Invocation` non-null? How about `run.Invocation?.ToolNotifications`?